pkg/storage/ingest/fetcher.go

```diff
@@ -226,14 +229,8 @@ type concurrentFetchers struct {
 
 	// trackCompressedBytes controls whether to calculate MaxBytes for fetch requests based on previous responses' compressed or uncompressed bytes.
 	trackCompressedBytes bool
-}
-
-// Stop implements fetcher
```

Comment:
This is just a move as is - these methods above the constructor seem to be oddly placed.